### PR TITLE
remove LD flag for custom code logging

### PR DESCRIPTION
**Pull request by byang-confluent**
Release Notes
-------------

Breaking Changes
- PLACEHOLDER

New Features
- PLACEHOLDER

Bug Fixes
- PLACEHOLDER

Checklist
---------
- [ ] Leave this box unchecked if features are not yet available in production

What
----

References
----------

Test & Review
-------------

### CODE REVIEW

File: internal/command.go [hunk 1]
```diff
@@ -139,9 +140,6 @@ func NewConfluentCommand(cfg *config.Config) *cobra.Command {
 	if cfg.IsTest || featureflags.Manager.BoolVariation("cli.flink", cfg.Context(), config.CliLaunchDarklyClient, true, false) {
 		cmd.AddCommand(flink.New(cfg, prerunner))
 	}
-	if cfg.IsTest || featureflags.Manager.BoolVariation("custom-connect.cli.custom_code_logging.early_access", cfg.Context(), config.CliLaunchDarklyClient, true, false) {
```
Comment (byang-confluent) [inline internal/command.go:142]: custom-connect.x flag should not working with cli. remove this flag from CLI

Comment (byang-confluent) [reply internal/command.go:142]: we have backend flag to control this feature. and this feature is OK to expose to all customers

Comment (sgagniere) [reply internal/command.go:142]: @byang-confluent Do we also want to expose the command reference documentation with this change?

Comment (byang-confluent) [reply internal/command.go:142]: @sgagniere , the public doc is not ready yet, we have ticket to track https://confluentinc.atlassian.net/browse/DOCS-31145

File: internal/command.go [hunk 2]
```diff
@@ -131,6 +131,7 @@ func NewConfluentCommand(cfg *config.Config) *cobra.Command {
 	cmd.AddCommand(streamshare.New(prerunner))
 	cmd.AddCommand(update.New(cfg, prerunner))
 	cmd.AddCommand(version.New(prerunner, cfg.Version))
+	cmd.AddCommand(ccl.New(cfg, prerunner))
```
Comment (sgagniere) [inline internal/command.go:134]: Please alphabetize this list.

Otherwise, lgtm

Comment (byang-confluent) [reply internal/command.go]: move it under `connect` for `custom-code-logging`

**Review by sgagniere — APPROVED** [review]

**Pull request merged**